src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -1109,6 +1159,8 @@ private static String addPrefix(String server) {
      * See http://msdn.microsoft.com/en-us/library/aa746475(VS.85).aspx for the syntax by example.
      * WANTED: The specification of the syntax.
      */
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", 
+            justification = "Filter may be changed in the runtime by scripts if required")
```

Comment:
If somebody is trying to write to this field from the groovy console (which is the only place you would try now) then they can stop. If they were trying to write to it from `init.groovy`, then they should stop. Make the field final to help them